packages/qdrant-loader-core/src/qdrant_loader_core/graph/falkor_store.py

```
@@ -0,0 +1,373 @@
+                    MATCH (a {{id: e.source}})
+                    MATCH (b {{id: e.target}})
+                    MERGE (a)-[r:{edge_type}]->(b)
+                    SET r += e.props
+                    """,
+                        {"edges": without_project},
+                    )
+                )
+        if tasks:
+            await asyncio.gather(*tasks)
+
+    async def neighbors(
```

Ruff (0.15.20)

[warning] Too many statements (61 > 50)

(PLR0915)